tests/test_ucm.py

```diff
@@ -69,7 +69,7 @@ def test_ucm_regression_factors(self):
             'avg_cc': 0.222150472947109,
             'avg_tmp_v': 37.325275675470998,
             'avg_tmp_an': 2.325275675470998,
-            'avd_eng_cn': 3520213.280928277,
+            'avd_eng_cn': 3520217.313878,

```

Comment:
Are these values changing because of a change in how we are masking?  If the behavior is changing, I wonder if we should note that change in HISTORY

Reply:
These had to be updated because of changes to zonal_statistics. Out of almost 1500 building polygons in the test data, zonal stats results were different (and more correct) for 7 of them. I am guessing because we used to reproject the raster to match the vector; now we reproject the vector to match the raster, which should be more accurate. This probably only showed up in UCM because it uses so many polygons and they are small relative to the pixel size.

Reply:
I added a general note in history.